src/setup/extensions/intermediateMessageCatchEvent.js

```diff
@@ -0,0 +1,26 @@
+import {
+  intermediateMessageCatchEvent,
+} from '@/components/nodes';
+import webHookFormGroup from '../../components/nodes/intermediateMessageCatchEvent/webHookFormGroup.vue';
+
+window.ProcessMaker.EventBus.$on('modeler-init', ({ registerInspectorExtension }) => {
+
+  /* Add custom properties to inspector */
+  registerInspectorExtension(intermediateMessageCatchEvent, {
+    id: 'pm-web-hook',
+    component: webHookFormGroup,
+    config: {
+      name: 'pm:webHook',
+    },
+  });
+
+  registerInspectorExtension(intermediateMessageCatchEvent, {
```

Comment:
Why not add this to the config in `src/components/nodes/intermediateMessageCatchEvent/index.js`?

Reply:
Had a discussion with @caleeli and he thought it would be best to add the two attributes as an extension since they are not part of the bpmn spec.